### PR TITLE
Add apache translations

### PR DESCRIPTION
**Pull request by jjbuchan**
Translations for this model - https://github.com/racker/salus-telemetry-monitor-management/pull/126/files#diff-1ac2c45abbe7e2fc3e53632738290703R33-R44

to convert to this telegraf plugin - https://github.com/influxdata/telegraf/tree/master/plugins/inputs/apache

### CODE REVIEW

File: monitor-translations/apache/create-urls-list.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "agentType": "TELEGRAF",
+  "monitorType": "http",
```
Comment (itzg) [inline monitor-translations/apache/create-urls-list.json]:
```suggestion
  "monitorType": "apache",
```

File: monitor-translations/apache/create-urls-list.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "name": "create-urls-list",
```
Comment (itzg) [inline monitor-translations/apache/create-urls-list.json]: Since name needs to be unique across all these, how about
```suggestion
  "name": "apache-create-urls-list",
```

File: monitor-translations/apache/rename-timeout.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "agentType": "TELEGRAF",
+  "monitorType": "apache",
+  "name": "rename-timeout",
```
Comment (itzg) [inline monitor-translations/apache/rename-timeout.json]: and
```suggestion
  "name": "apache-rename-timeout",
```

**Review by itzg — CHANGES_REQUESTED** [review]

File: monitor-translations/apache/create-urls-list.json (hunk 1 above)
Comment (jjbuchan) [reply monitor-translations/apache/create-urls-list.json]: huh... thought i got all those

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,2 @@
-*/.idea/
+*.idea/
```
Comment (itzg) [inline .gitignore:1]: Just curious why this change

**Review by itzg — APPROVED** [review]

File: .gitignore (hunk 4 above)
Comment (jjbuchan) [reply .gitignore:1]: Because a `git status` shows this

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	.idea/
```

I think when I initially made the change to add this I had only opened up the `monitor-translations` folder so it created an .idea in there and that led to `*/.idea/` working.  Now that I've opened up the main `salus-data-loader-content` folder the .idea at the top level isn't caught by the original ignore.

**Review by harageth — APPROVED** [review]
